tests/test_functional.py

```diff
@@ -657,19 +657,16 @@ def test_obal_changelog_custom():
 @obal_cli_test(repotype='upstream')
 def test_obal_srpm():
     assert_obal_success(['srpm', 'hello', 'foo'])
-    assert_mockbin_log([
-        'tito build --srpm --offline --output {pwd}/SRPMs',
-        'tito build --srpm --offline --output {pwd}/SRPMs'
-    ])
+
+    assert os.path.exists('SRPMs/foo-1.0-1.src.rpm')
+    assert os.path.exists('SRPMs/hello-2.10-2.src.rpm')
 
 
 @obal_cli_test(repotype='upstream')
 def test_obal_mock():
     assert_obal_success(['mock', 'hello', 'foo', '--config', 'mock/el7.cfg'])
 
     expected_log = [
-        "tito build --srpm --offline --output {pwd}/SRPMs",
-        "tito build --srpm --offline --output {pwd}/SRPMs",
-        "mock --recurse --chain -r mock/el7.cfg --localrepo {pwd}/mock_builds SRPMs/foo-1.0-1.src.rpm SRPMs/hello-2.10-1.src.rpm"
+        "mock --recurse --chain -r mock/el7.cfg --localrepo {pwd}/mock_builds SRPMs/foo-1.0-1.src.rpm SRPMs/hello-2.10-2.src.rpm"
```

Comment:
why does the revision change from 1 to 2?

or rather: why did it work before!? as the testrepo clearly has -2…

Reply:
Maybe just how the expectation and the test wrapper were handling things it masked that it didn't matter what was passed? I really don't know lol. The spec in the fixtures has a `2` for it's release.